src/index.js

```diff
@@ -129,13 +130,14 @@ export function filterCards(cards, options) {
 }
 
 export function deckFromCards(cards, images, options) {
+	// get path to highlight.js cdn directory
+	let cssPath = path.dirname(resolve.sync('@highlightjs/cdn-assets/package.json'));
```

Comment:
We need to add the `basedir`. Otherwise it fails on global install:

```suggestion
	let cssPath = path.dirname(resolve.sync('@highlightjs/cdn-assets/package.json', { basedir: __dirname }));
```

Reply:
Oh neat, so I almost got it right. Thanks for the correction!